src/reference/informative/informC02_units_reference.rst

```diff
@@ -8,4 +8,300 @@
+
+    In order to reuse the :code:`spoonful` units from either of the imported models, they must be explicitly imported.  The top-level model becomes:
+
+    .. code::
+
+        model: BlueberryPieRecipe
+          ├─ component: BrandyCustard
+          │    ├─ variable: custard (litre)
+          │    └─ variable: brandy           
+      ┌╴╴╴╴╴╴╴╴> └─ units: spoonful
+      ╷   ├─ component: crust <╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴┐
+      ╷   ├─ component: filling <╴╴╴╴╴╴╴╴╴╴╴╴╴╴╴┐ ╷
+      ╷   │                                     ╷ ╷
+      └╴╴╴└─ units: spoonful <╴╴╴╴╴╴╴┐     imported components and
+                                     ╷     the units they need
+                  explicitly imported units     ╵ ╵
+            are available to all components     ╵ ╵
+                                     ╵          ╵ ╵
+      # In filling_recipes.cellml:   ╵          ╵ ╵
+      model: FillingRecipeCollection ╵          ╵ ╵
+        ├─ units: spoonful ╴╴╴╴╴╴╴╴╴╴┘          ╵ ╵
+        ├─ component: BlueberryCinnamonFilling ╶┘ ╵
+        └─ component: AppleAndPearFilling         ╵ 
+                                                  ╵
+      # In crust_recipes.cellml:                  ╵
+      model: PieCrustRecipes                      ╵
+        ├─ component: HazelnutLavenderCrust ╴╴╴╴╴╴┘
+        └─ component: CheeseAndAlmondCrust
+
+    .. container:: toggle
+
+      .. container:: header
+
+        See CellML syntax
+
+      .. code-block:: xml
+
+        <!-- Inside the file "how_to_make_blueberry_pie.cellml": -->
+        <model name="BlueberryPieRecipe">
+          <import xlink:href="path/to/my/crust_recipes.cellml">
+            <component name="premade_crust" component_ref="HazelnutLavenderCrust" />
+          </import>
+          <import xlink:href="path/to/my/filling_recipe.cellml">
+            <component name="yummy_filling" component_ref="BlueberryCinnamonFilling" />
+          </import>
+
+          <!-- Defining a new component, brandy custard -->
+          <component name="BrandyCustard">
+            <variable name="custard" units="litre" />
+            <variable name="brandy" units="spoonful" />
+            ...
+          </component>
+
+          <!-- Explicitly importing the "spoonful" units from the "filling_recipes.cellml" file: -->
+          <import xlink:href="path/to/my/filling_recipe.cellml">
+            <!-- The units are also called "spoonful" in this top-level scope. -->
+            <units name="spoonful" component_ref="spoonful" />
+          </import>
+        </model>
+              
+    At this stage we have three sets of units all named "spoonful".
+    Since each is only accessible to its local components there is no conflict of definition or interpretation.
+    Now that the units required in the new :code:`BrandyCustard` component are defined within the same infoset, the model becomes valid, and our dessert needs are satisfied once more.
```

Comment:
I am not sure I would use the word "infoset". This is too XML-centric while here I would expect the use of a more informal vocabulary? What about "scope"?

Reply:
895313c